src/adapters/adapter.utils.ts

```diff
@@ -0,0 +1,14 @@
+import Boom from 'boom';
+import { AdapterErrors } from './adapter.errors';
+import { defaultAdapter } from './default.adapter';
+import type { ADAPTERS } from './adapter.types';
+
+export const getAdapter = (adapterName: string) => {
```

Comment:
Aktualny task nie zakładał podłączenia się jeszcze pod adaptery. A stworzenie funkcjonalności które będą potrafiły manipulować multimediami.

Reply:
Tak jak wyżej pisałem, zrobiłem tylko taki dummy adapter, żeby móc przetestować czy to w ogóle będzie działać. Nic więcej w adapterach nie zamierzałem robić. Wyrzucę to w kolejnej PRce jak już zrobię setup testów i nie będę potrzebował klikać ręcznie tego